streaming.md

```diff
@@ -77,9 +83,13 @@ The role of the `StartFrame` is to carry information to the invoker that is rele
 Such information is made of:
 
 * *expectedContentTypes*: a list of N strings, where N is the number of output parameters of the function. Each string is a comma separated list of MIME types and indicates, in order, the preference of the streaming processor when it comes to receiving output data: if the i-th string is equal to `MIME-Type1, MIME-Type2` then every `OutputFrame` pertaining to the i-th output (that is, where `resultIndex == i`) MUST have its `contentType` field set to a MIME type that is either `MIME-type1` or `MIME-type2` (or a compatible subtype thereof).
+* *inputNames*: a list of M strings, where M is the number of input parameters of the function. Each string a logical stream name.
```

Comment:
The strings are *not* stream names. They are names of constructs in the target language (eg field names, map keys, etc). Chances are they will also be chosen as __logical role__ names (ie what that parameter does), but they are *not* stream names. It will be riff / the processor concept that will assign a mapping between stream and input name. Their default valuation will be done from the bound stream name, which is something entirely different.

Reply:
I'm not sure I understand what the target language has to do what input and output names are. As I understand it, the way they are exposed is language-specific, not what they are.

What about: 
```
Each string is a name mapped to an underlying stream resource.
```